ref-schema-url/test/fixtures/invalid-certificate-9.json

```diff
@@ -0,0 +1,3 @@
+{
+  "RefSchemaUrl": "https://schemas.s1seven.co.uk/en10168-schemas/v0.3.0/schema.json"
+}
```

Comment:
but this one should work ?

Reply:
The url has `.co.uk` instead of `.com`. We only support `.com` right?